src/views/AlertTable/i18n.json

```
@@ -0,0 +1,13 @@
+{
+    "namespace": "emergencies",
+    "strings": {
+        "allOngoingAlertTitle":"ALL ALERTS ",
```

```suggestion
        "allOngoingAlertTitle":"All Alerts ",
```
Making it similar to IFRC GO.